api/graphql/schema.graphql

```diff
@@ -398,74 +406,77 @@ enum EducationStatus {
 }
 
 interface Experience {
-  id: ID!
-  applicant: Applicant! @belongsTo(relation: "user")
-  details: String
-  experienceSkills: [ExperienceSkill] @morphMany
+    id: ID!
+    applicant: Applicant! @belongsTo(relation: "user")
+    details: String
+    skills: [Skill!] @morphToMany
+    experienceSkillRecord: ExperienceSkillRecord @rename(attribute: "pivot")
```

Comment:
I haven't tested these queries yet in the playground, but if I understand correctly the pivot record is only available in certain contexts. ie when Experience is nested inside Skill. 
eg `{skill { experiences { experienceSkillRecord { detail }}}}` would be valid but `{user {experiences {experienceSkillRecord {detail }}}}` would not.
I'm not sure what the best way around this is. If nothing else, can we leave a comment in the schema explaining this?
This applies equally to other instances of experienceSkillRecord fields.

Reply:
Yeah, that's how it works (if you try to query for it directly detail just returns null) but I'm also unsure of any good way around this. I'll add a comment explaining this now though.